src/engraving/dom/spanner.h

```diff
@@ -206,7 +208,9 @@ class Spanner : public EngravingItem
     virtual void undoChangeProperty(Pid id, const PropertyValue&, PropertyFlags ps) override;
 
     void computeStartElement();
+    virtual void doComputeStartElement();
```

Comment:
It looks like these should be protected. Also, what's the difference from computeStart/EndElement? Maybe they should be virtual instead?

Reply:
computeEndElement() is the part that is common to all Spanners, doComputeEndElement() is the part that can be overridden (and yes, it should be protected, I've corrected it now)